CONTRIBUTING.md

```
@@ -226,34 +226,50 @@ To maintain a clean and linear project history, we prefer rebasing your feature
 
+
 2.  **Rebase your feature branch**:
 
     ```bash
     git checkout your-feature-branch
     git rebase origin/develop
     ```
 
+
+    > **💡 GUI Alternative**: In VSCode, ensure you're on your feature branch, then open the Command Palette (Ctrl/Cmd+Shift+P), type "Git: Rebase Branch" and select `origin/develop` as the target.
+
```

markdownlint-cli2 (0.17.2)

Blank line inside blockquote

(MD028, no-blanks-blockquote)